### ISSUE TITLE
Tailwind v3 neutral color palette no longer working after installing daisyui

### ISSUE THREAD

**Issue by sbahman** [saadeghi/daisyui#1070]
Tailwind added the neutral color palette for v3 [https://tailwindcss.com/docs/customizing-colors](https://tailwindcss.com/docs/customizing-colors). After installing daisyui there is a conflict as daisyUI also defines a neutral color.

This can be fixed with the following addition to the tailwind.config as described in this [StackOverflow post](https://stackoverflow.com/questions/73502213/tailwindcss-neutral-500-color-no-longer-working-after-installing-daisyui/73528668?noredirect=1#comment129847364_73528668)

```
const colors = require('tailwindcss/colors');

/** @type {import('tailwindcss').Config} */
module.exports = {
    content: [],
    theme: {
        extend: {
            colors: {
                neutral: colors.neutral,
            }
        }
    },
    plugins: [
        require("daisyui")
    ],
}
```

If possible I think this should not be necessary or else this should be mentioned in the docs.

**Comment by shrekuu** [comment]
I also find my neutral color does not work now.
eg: `bg-neutral-300`  

I got `neutral`, `neutral-focus`, `neutral-content` these 3 colors. 

<img width="492" alt="image" src="https://user-images.githubusercontent.com/3136026/188042734-063a26ca-270e-4348-b34a-c7035ebcf83d.png">

**Comment by saadeghi** [comment]
This is the reason why it's not working: https://github.com/saadeghi/daisyui/issues/683#issuecomment-1086754993

I will try to find a way to fix this without creating duplicate styles...

**Comment by ben-xD** [comment]
Extra note: This also happens with `primary`, probably for the same reason. I agree that "semantic color names" (`primary`) is nicer than using actual colors - that's why I'm using it even with tailwind (before I heard of DaisyUI). It's not nice that my colors are broken. 🤓😅

Side note: I think it would be fine if DaisyUI overrides them, but currently it doesn't look like there is a way to configure the color **palette** (50, 100, 200, ..., 800, 900).

DaisyUI was really nice to use today in some experimentation. Thanks :)

**Comment by ylor** [comment]
Was also bitten by this.

I would also agree that daisyUI "should" (rather than needs to) follow Tailwind defaults and enhance/extend.

**Comment by ylor** [comment]
Was also bitten by this.

I would also suggest that daisyUI, as a Tailwind library, should only extend/enhance Tailwind's defaults.

**Comment by saadeghi** [comment]
Fixed in latest version `2.49.0`
